src/libs/fileDownload/index.ts

```diff
@@ -11,7 +11,12 @@ import type {FileDownload} from './types';
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const fileDownload: FileDownload = (url, fileName, successMessage = '', shouldOpenExternalLink = false) => {
     const resolvedUrl = tryResolveUrlFromApiRoot(url);
-    if (shouldOpenExternalLink || (!resolvedUrl.startsWith(ApiUtils.getApiRoot()) && !CONST.ATTACHMENT_LOCAL_URL_PREFIX.some((prefix) => resolvedUrl.startsWith(prefix)))) {
+    if (
+        shouldOpenExternalLink ||
+        (!resolvedUrl.startsWith(ApiUtils.getApiRoot()) &&
+            !CONST.ATTACHMENT_LOCAL_URL_PREFIX.some((prefix) => resolvedUrl.startsWith(prefix)) &&
+            url !== CONST.EXPENSIFY_PACKAGE_FOR_SAGE_INTACCT)
```

Comment:
Why is this needed?

Reply:
Clicking 'Download the Expensify package for Sage Intacct' should download the link on press. Without adding that condition to `fileDownload.ts` this download action will be blocked because it's am external link.
<img width="379" alt="Screenshot 2024-06-27 at 11 46 17" src="https://github.com/Expensify/App/assets/88395093/21e1eea6-c71b-4173-9a9f-63f3803cf819">